ckanext/archiver/commands.py

```diff
@@ -632,11 +636,20 @@ def delete_files_larger_than_max_content_length(self):
                 model.Session.flush()
                 continue
             filepath = archival.cache_filepath
-            if not os.path.exists(filepath):
+            if not filepath.startswith('http') and not os.path.exists(filepath):
                 print 'Skipping - file not on disk'
                 continue
             try:
-                os.unlink(filepath)
+                if filepath.startswith('http'):
+                    # Uploader system used
+                    relative_archive_path = os.path.join('archive', archival['resource_id'][:2],
+                                                         resource['resource_id'])
+                    filename = os.path.basename(filepath)
```

Comment:
This isn't an appropriate way to handle a URL with protocol. See the test failure for an example of the result.